src/domain/app/i18n/fi.json

```diff
@@ -521,6 +521,8 @@
     "pageTitle": "Tapahtuma-aika",
     "textDateAndTime": "{{date}} klo {{time}}",
     "textAmountOfSeats": "{{count}} paikka",
+    "textAmountOfGroups": "{{count}} ryhmä",
+    "textAmountOfGroups_plural": "{{count}} ryhmää",
```

Comment:
I did not find that `textAmountOfGroups_plural` is used anywhere?

Reply:
_plural works automatically when count is  0 or more than 1

see: https://www.i18next.com/translation-function/plurals

Reply:
Nice one, I did not know